### PR TITLE
docs: add `Menu.More` translation

### PR DESCRIPTION
**Pull request by stoffeastrom**
## Motivation

Add translation for `Menu.More`

### CODE REVIEW

**Comment by stoffeastrom** [comment]
@Linguisht Is this enough e.g will the automation (e.g you manually :)) also take care of filling in the `comment` ❓

File: apis/locale/locales/en-US.json [hunk 1]
```diff
@@ -23,6 +23,10 @@
     "value": "Search in listbox",
     "comment": "Action text to search in listbox"
   },
+  "Menu.More": {
+    "value": "More",
+    "comment": "$"
```
Comment (Linguisht) [inline apis/locale/locales/en-US.json:28]: Suggested comment: `Menu option to display list of additional actions to choose from (ffd 20200520)`

Going forward I checked with the tech writer team and my understanding is that Dan Chabot (tech writer) will be available to review strings. I'll check directly with him and ask him to join this org.

Comment (stoffeastrom) [reply apis/locale/locales/en-US.json:28]: Maybe we should add that to the documentation https://github.com/qlik-oss/nebula.js/blob/master/apis/locale/locales/README.md

Comment (stoffeastrom) [inline apis/locale/locales/en-US.json:28]:
```suggestion
    "comment": "Menu option to display list of additional actions to choose from (ffd 20200520)"
```

**Comment by stoffeastrom** [comment]
FYI For fast changes you can comment with [suggestions](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request).

File: apis/locale/locales/en-US.json (hunk 1 above)
Comment (Linguisht) [reply apis/locale/locales/en-US.json]: Yes, I plan to update the readme next week, to capture everything ([RDL-3536](https://jira.qlikdev.com/browse/RDL-3536))

**Pull request merged**